cbopensource/connectors/taxii/config_util.py

```diff
@@ -120,10 +131,11 @@ def parse_config(config_file_path):
         #
         # Added the ability to limit the number of reports per collection
         #
+
         if config.has_option(section, 'reports_limit'):
             reports_limit = config.getint(section, 'reports_limit')
         else:
-            reports_limit = 10000
+            reports_limit = default_reports_limit
```

Comment:
If there already is a reports_limit, I'm not sure what the new functionality the config option is giving to the user.

Reply:
It's a connector-level default instead of a site-level default for the number of reports pulled.